File: app/src/main/java/com/androidevlinux/percy/UTXO/data/models/changelly/ParamsBean.java
package com.androidevlinux.percy.UTXO.data.models.changelly;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by percy on 15/11/2017.
 */

public class ParamsBean {

    @SerializedName("from")
    @Expose
    private String from;
    @SerializedName("to")
    @Expose
    private String to;
    @SerializedName("amount")
    @Expose
    private String amount;
    @SerializedName("address")
    @Expose
    private String address;
    @SerializedName("id")
    @Expose
    private String id;

    public String getFrom() {
        return from;
    }

    public void setFrom(String from) {
        this.from = from;
    }

    public String getTo() {
        return to;
    }

    public void setTo(String to) {
        this.to = to;
    }

    public String getAmount() {
        return amount;
    }

    public void setAmount(String amount) {
        this.amount = amount;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }
}

File: app/src/main/java/com/androidevlinux/percy/UTXO/data/models/changelly/ParamsBean.kt
package com.androidevlinux.percy.UTXO.data.models.changelly

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by percy on 15/11/2017.
 */

class ParamsBean {

    @SerializedName("from")
    @Expose
    var from: String? = null
    @SerializedName("to")
    @Expose
    var to: String? = null
    @SerializedName("amount")
    @Expose
    var amount: String? = null
    @SerializedName("address")
    @Expose
    var address: String? = null
    @SerializedName("id")
    @Expose
    var id: String? = null
}
